api/src/main/kotlin/com/google/devtools/ksp/processing/FileType.kt

```diff
@@ -0,0 +1,12 @@
+package com.google.devtools.ksp.processing
+
+/**
+ * Used to determine where files should be stored when being
+ * created through the [CodeGenerator].
+ */
+enum class FileType {
```

Comment:
This `FileType` should be unnecessary then? I see that you do have usages in your code relying on this enum, but since we do not have any APIs exposing this enum, maybe we can put it in our implementations, if this enum helps with your implementation.

Reply:
Yeah, got rid of it